lib/platforms/windows.dart

```diff
@@ -54,11 +54,23 @@ void _setWindowsAppTitle(String appName) {
     }
 
     final mainCppString = mainCppFile.readAsStringSync();
-    final newAppTitleMainCppString = mainCppString.replaceAll(
+
+    var newAppTitleMainCppString = mainCppString.replaceAll(
       RegExp(r'window.CreateAndShow\(L"(.*?)"'),
       'window.CreateAndShow(L"$appName"',
     );
 
+    // Flutter version 3.7.7 changed the way how a window is created
+    // to the following code in main.cpp:
+    //
+    // if (!window.Create(L"my_old_app_name", origin, size)) {
+    //   return EXIT_FAILURE;
+    // }
+    newAppTitleMainCppString = newAppTitleMainCppString.replaceAll(
+      RegExp(r'window.Create\(L"(.*?)"'),
+      'window.Create(L"$appName"',
+    );
```

Comment:
Can you try cascading this code?

Reply:
Resolved in [new commit](https://github.com/obemu/package_rename/commit/23fccf09eeb92780e0bd0e10b45c231503e02128)